### PR TITLE
[Chore] Restore e2e test

### PR DESCRIPTION
**Pull request by Atralupus**
Add "UseUpdate" option in `IConfig`

### CODE REVIEW

File: src/main/update/launcher-update.ts [hunk 1]
```diff
@@ -69,6 +69,10 @@ export interface IUpdateOptions {
 }
 
 export async function update(update: Update, listeners: IUpdateOptions) {
+  if (!getConfig("UseUpdate", true)) {
```
Comment (BasixKOR) [inline src/main/update/launcher-update.ts:72]:
```suggestion
  if (!getConfig("UseUpdate", process.env.NODE_ENV === "production")) {
```

Just a touch to disable updates during development.

File: src/main/update/launcher-update.ts [hunk 2]
```diff
@@ -69,6 +69,10 @@ export interface IUpdateOptions {
...
+    return;
```
Comment (BasixKOR) [inline src/main/update/launcher-update.ts:73]: Consider adding a log here.

Comment (Atralupus) [reply src/main/update/launcher-update.ts:73]: Good

File: src/main/update/launcher-update.ts (hunk 1 above)
Comment (Atralupus) [reply src/main/update/launcher-update.ts:72]: Gooooood

**Review by BasixKOR — APPROVED** [review]

**Pull request merged**